src/config.js

```diff
@@ -37,16 +37,24 @@ const config = (userConfig = {}) => {
     },
     userConfig
   );
-  if (!newConfig.purge) {
-    newConfig.purge = [purge];
-  } else if (Array.isArray(newConfig.purge)) {
-    newConfig.purge = [...newConfig.purge, purge];
-  } else if (newConfig.purge.content) {
-    newConfig.purge.content = [...newConfig.purge.content, purge];
-  } else if (!newConfig.purge.content) {
-    newConfig.purge.content = [purge];
```

Comment:
I think we don't need to support v2 anymore. Here move back this block with the following replacements to match v3 API:
- `newConfig.purge` -> `newConfig.content`
- `newConfig.purge.content` -> `newConfig.content.files` (https://tailwindcss.com/docs/content-configuration#transforming-source-files)

Reply:
also renamed `flex-shrink` to `shrink`